zerver/views/streams.py

```diff
@@ -206,13 +209,13 @@ def compose_views(request, user_profile, method_kwarg_pairs):
     return json_success(json_dict)
 
 @has_request_variables
-def remove_subscriptions_backend(request, user_profile,
-                                 streams_raw = REQ("subscriptions", validator=check_list(check_string)),
-                                 principals = REQ(validator=check_list(check_string), default=None)):
-    # type: (HttpRequest, UserProfile, Iterable[Text], Optional[Iterable[Text]]) -> HttpResponse
+def remove_subscriptions_backend(
+        request: HttpRequest, user_profile: UserProfile,
+        streams_raw: Iterable[Text]=REQ("subscriptions", validator=check_list(check_string)),
+        principals: Optional[Iterable[Text]]=REQ(validator=check_list(check_string), default=None),
+) -> HttpResponse:
 
-    removing_someone_else = principals and \
-        set(principals) != set((user_profile.email,))
+    removing_someone_else = principals and set(principals) != set((user_profile.email,))
```

Comment:
What's the story behind this change? In any case, it doesn't seem related to the mypy annotations, so it should go into a separate commit?

Reply:
Good catch -- this is just a formatting change with no runtime effect, but I reverted it while merging to keep the commit focused.